Misc/NEWS.d/next/Core and Builtins/2022-02-07-17-40-23.bpo-46671.CmpNlQ.rst

```diff
@@ -0,0 +1 @@
+Change exception message of :func:`min()` and :func:`max()` to refer to an iterable rather than a sequence if an empty iterable of values to select from was provided.
```

Comment:
```suggestion
Change exception message of :func:`min` and :func:`max` to refer to an iterable rather than a sequence if an empty iterable was provided.
```
Fix ReST link and make the text more concise.

Reply:
I had a purpose with this. `min()`/`max()` have different ways of being parametrized. Iterables are objects just like any other, and e.g. providing several iterables would cause the function to try to pick the maximum / minimum of the iterable objects, according to how their comparison operators are defined. I wanted to avoid insinuating that empty iterables in any other context than being the single iterable of values to choose the minimum / maximum from are somehow treated as an error.